fireblocks_sdk/api_types.py

```diff
@@ -262,11 +262,38 @@ def __init__(self, total_amount_larger_than=None, asset_id=None, order_by=None,
         self.after = after
 
 
-class NFTOwnershipStatusValues(Enum):
+class GetOwnedNftsSortValues(str, Enum):
```

Comment:
Please make sure this doesn't break backwards compatibility

Reply:
It doesn't (actually the usage was wrong until now)